src/app/views/experiment_page.js

```diff
@@ -0,0 +1,68 @@
+import View from 'ampersand-view';
+import dom from 'ampersand-dom';
+import mustache from 'mustache';
+import webChannel from '../lib/web_channel';
+
+export default View.extend({
+  template: mustache.render(`<section class="page" data-hook="experiment-page">
```

Comment:
Well, the "Ampersand way" of rendering lists is to use [a collection](https://github.com/lmorchard/c25k-web/blob/master/src/js/models/WorkoutCollection.js) of [model/state objects](https://github.com/lmorchard/c25k-web/blob/master/src/js/models/Workout.js). a [per-item subview](https://github.com/lmorchard/c25k-web/blob/master/src/js/views/WorkoutListItem.js), and [`renderCollection` in a parent view](https://github.com/lmorchard/c25k-web/blob/master/src/js/views/WorkoutList.js). 

You don't *have* to do it that way, of course. The main benefit you get from it, though, is you can set event listeners on each of the subviews and have them auto-update when the underlying model changes. That might not be necessary here.